### ISSUE TITLE
FTB Genesis Combustion Chamber failing to produce power

### ISSUE THREAD

**Issue by SgtKrazy** [FTBTeam/FTB-Modpack-Issues#5202]
### Modpack

FTB Genesis

### Modpack version

1.5.0

### Describe the bug

Placing a combustion chamber to produce volts does not produce volts.

### Steps to reproduce

1. Place a combustion chamber
2. Place a Oil tank (Creative Crude oil or non creative crude oil)
3. Connect tank to combustion chamber with fluid transporter
4. Place a LV Flux converter with a minimum 1 block space
5. In 1 the 1 block space between chamber and converter place a copper wire
6. Place a electric furnace 1 block away from LV converter
7. In the 1 block between electric furnace and LV converter place a insulated copper cable

### Expected behaviour

Electric Furnace should have power

### Screenshots

![2024-05-08_23 40 47](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/21181138/c684b728-d2dc-4774-a2c3-30e9810bc12a)
![2024-05-08_23 50 37](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/21181138/c9e37ae0-c609-43d6-ac88-454335bcd253)
![2024-05-08_23 50 43](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/21181138/561e2b02-18fe-4eed-8d58-e0e88fa40577)
![2024-05-08_23 50 50](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/21181138/33f44624-72d8-4d85-9779-4044eb875680)

### Additional information

This is on dedicated server and single player set up. Both are new settings.

**Comment by CarterJWallace** [comment]
Looks to be a general change where electrodynamics power can not be directly input into IC2 flux any longer. Was possible in 1.4.0. Now you have to put Mekanism in the middle.

**Comment by SgtKrazy** [comment]
It does work with adding Mekanism in the middle but, on restart of server it fails to work sometimes.

**Comment by ArtpokeBr** [comment]
Yeah this seems to be a bug with ElectroDynamics, but I found out that connecting directly to block faces works. only the wires are being affected.

![ed_converting_bug](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/44851727/360425fd-5adb-4be4-8d78-a587a9ea1aa0)

**Comment by manmaed** [comment]
https://github.com/aurilisdev/Electrodynamics/issues/258

**Comment by manmaed** [comment]
Tested in  1.6.1
